src/main/python/trec-covid/covid_baseline_tools.py

```diff
@@ -24,6 +24,146 @@
 import pyserini.util
 
 
+def perform_runs(round_number, indexes):
+    base_topics = f'src/main/resources/topics-and-qrels/topics.covid-round{round_number}.xml'
+    udel_topics = f'src/main/resources/topics-and-qrels/topics.covid-round{round_number}-udel.xml'
+
+    # Use cumulative qrels from previous round for relevance feedback runs
+    cumulative_qrels = f'src/main/resources/topics-and-qrels/qrels.covid-round{round_number - 1}-cumulative.txt'
```

Comment:
will this break if round_number=1?

Reply:
Correct. This will only work for rounds 3, 4, 5 - where we've established a consistent set of runs. It'll most definitely break for rounds 1 and 2. I'll need to go back and retro fit those runs to make consistent.

Reply:
All right, I will approve it now so it will be up to you if you want to do this in this PR or in another one